src/client/StatusModal.jsx

```diff
@@ -81,10 +83,9 @@ export default class StatusModal extends React.Component {
   }
 
   getMessage () {
-    if (this.props.item.status !== 'finished') {
-      return 'Game is not finished'
+    if(this.props.item.status === 'pendning' || this.props.item.status === 'waiting for opponent'){
```

Comment:
pendning => pending

Reply:
Btw, here common.gameStatua should be used I guess... I must have missed that before.

Reply:
Fixed the spelling error. Perhaps the refactoring to common.gameStatus should be done in a separate PR, as it's not related to the rematch?